Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1488,10 +1544,70 @@ bool applyTexture(
   return true;
 }
 
+static void loadPointCloud(
+    UCesiumGltfComponent* pGltf,
+    LoadModelResult& loadResult,
+    const glm::dmat4x4& cesiumToUnrealTransform) {
+  if (!loadResult.pointCloudBuffer) {
+    return;
+  }
+
+  UCesiumPointCloudComponent* pMesh = NewObject<UCesiumPointCloudComponent>(
+      pGltf,
+      FName(loadResult.name.c_str()));
+  pMesh->HighPrecisionNodeTransform = loadResult.transform;
+  pMesh->UpdateTransformFromCesium(cesiumToUnrealTransform);
+
+  // pMesh->bUseDefaultCollision = false;
+  // pMesh->SetCollisionObjectType(ECollisionChannel::ECC_WorldStatic);
+  pMesh->SetFlags(
+      RF_Transient | RF_DuplicateTransient | RF_TextExportTransient);
+  pMesh->Metadata = std::move(loadResult.Metadata);
+  pMesh->pModel = loadResult.pModel;
+  pMesh->pMeshPrimitive = loadResult.pMeshPrimitive;
+
+  ULidarPointCloud* pPointCloud =
+      NewObject<ULidarPointCloud>(pMesh, FName(loadResult.name.c_str()));
+
+  // TODO: try async load here
+  ELidarPointCloudAsyncMode asyncPointCloudResult =
+      ELidarPointCloudAsyncMode::Progress;
+  float progress = 0.0f;
+  ULidarPointCloudBlueprintLibrary::CreatePointCloudFromData(
+      pMesh,
+      *loadResult.pointCloudBuffer,
+      false,
+      FLatentActionInfo(),
+      asyncPointCloudResult,
+      progress,
+      pPointCloud);
+
+  pMesh->SetPointCloud(pPointCloud);
+  pMesh->PointSize = 1000.0f / pMesh->Bounds.SphereRadius;
+  pMesh->PointShape = ELidarPointCloudSpriteShape::Circle;
```

Comment:
This one apparently became `private` in 4.27, and this line then has to be

    pMesh->SetPointShape(ELidarPointCloudSpriteShape::Circle);